test/e2e/apps/deployment.go

```diff
@@ -1476,3 +1480,75 @@ func waitForDeploymentCompleteAndCheckRolling(c clientset.Interface, d *appsv1.D
 func waitForDeploymentUpdatedReplicasGTE(c clientset.Interface, ns, deploymentName string, minUpdatedReplicas int32, desiredGeneration int64) error {
 	return testutil.WaitForDeploymentUpdatedReplicasGTE(c, ns, deploymentName, minUpdatedReplicas, desiredGeneration, poll, pollLongTimeout)
 }
+
+//Deployment should have a working scale subresource
+func testDeploymentSubresources(f *framework.Framework) {
+	ns := f.Namespace.Name
+	c := f.ClientSet
+
+	deploymentName := "test-new-deployment"
+	podLabels := map[string]string{"name": WebserverImageName}
```

Comment:
Just move this directly to `NewDeployment` invocation, you don't need this variable so let's not create it :wink:

Reply:
looks like this is outstanding

Reply:
Updated it directly to `NewDeployment`
Thank you for the advice.